packages/@tinacms/cli/src/next/vite/index.ts

```diff
@@ -155,17 +155,22 @@ export const createConfig = async ({
     await fs.outputFile(staticMediaPath, `[]`)
   }
 
+  const isBrowser = typeof window !== 'undefined'
+
   const alias = {
     TINA_IMPORT: configManager.prebuildFilePath,
     SCHEMA_IMPORT: configManager.generatedGraphQLJSONPath,
     STATIC_MEDIA_IMPORT: staticMediaPath,
 
     // browser build only replacements
-    crypto: path.join(configManager.spaRootPath, 'src', 'dummy-client.ts'),
-    fs: path.join(configManager.spaRootPath, 'src', 'dummy-client.ts'),
-    os: path.join(configManager.spaRootPath, 'src', 'dummy-client.ts'),
-    path: path.join(configManager.spaRootPath, 'src', 'dummy-client.ts'),
+    ...(isBrowser && {
+      crypto: path.join(configManager.spaRootPath, 'src', 'dummy-client.ts'),
+      fs: path.join(configManager.spaRootPath, 'src', 'dummy-client.ts'),
+      os: path.join(configManager.spaRootPath, 'src', 'dummy-client.ts'),
+      path: path.join(configManager.spaRootPath, 'src', 'dummy-client.ts'),
+    }),
   }
+
   if (configManager.shouldSkipSDK()) {
```

Comment:
why are these changes needed? Isn't this only executed in the cli dev/build commands?

Reply:
I originally thought these could potentially be causing the issue as to why the import wasn't working. After testing this some more - I have noticed that isBrowser is always false. So Will remove these 🙏